core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5907,25 +5907,6 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_UFS_BLOCK_READ_LOCATION_POLICY_CACHE_SIZE =
```

Comment:
What's our policy for removing `PropertyKey`s? Should we deprecate them instead?